### PR TITLE
documentation for password options

### PR DESCRIPTION
**Pull request by CarolynRountree**
Documentation for command line options for passwords

### CODE REVIEW

File: documentation/1.9/content/userguide/tools/deploy.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To run the tool in online mode, add the `-admin_url` and `admin_user` arguments
 
     $ weblogic-deploy\bin\deployApps.cmd -oracle_home c:\wls12213 -domain_home domains\DemoDomain -archive_file DemoDomain.zip -model_file DemoDomain.yaml -variable_file DemoDomain.properties -admin_url t3://127.0.0.1:7001 -admin_user weblogic
 
-As usual, the tool will prompt for the password (it can also be supplied by piping it to standard input of the tool).
+As usual, the tool will prompt for the password (it can also be supplied by piping it to standard input of the tool). To bypass the prompt, you can use one of two options. Store the password in an environment variable, and use the variable name with command line option -admin_pass_env. Store the password as the only variable in a file. Provide the file name with command line option -admin_pass_file.
```
Comment (ddsharpe) [inline documentation/1.9/content/userguide/tools/deploy.md:29]: Replace `Store the password as the only variable in a file.` with the text that gets used from the previous comment.

File: documentation/1.9/content/userguide/tools/create.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ To create more complex domains, it may be necessary to create a custom domain ty
...
+If the model or variables file contains passwords encrypted with the WDT Encryption tool, decrypt the passwords during create with the `-use_encryption` flag on the command line to tell the Create Domain Tool that encryption is being used and to prompt for the encryption passphrase.  As with the database passwords, the tool can also read the passphrase from standard input (for example, `stdin`) to allow the tool to run without any user input. You can bypass the stdin prompt with two other options. Store the passphrase in an environment variable, and use the environment variable name with command line option -passphrase_env. Another option is to create a file with the single value of passphrase. Provide the name of the file with command line option -passphrase_file. The passphrase will be read by the tool from the file.
```
Comment (ddsharpe) [inline documentation/1.9/content/userguide/tools/create.md:35]: Another option is to create a file with the single value of passphrase. -> Another option is to create a file with the value for the passphrase. 
All similar sentences below should use the same text.   @rosemarymarano any suggestions?

File: documentation/1.9/content/userguide/tools/deploy.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ When running the tool in WLST online mode, the deploy operation may require serv
...
+If the model or variables file contains passwords encrypted with the WDT Encryption tool, decrypt the passwords during create with the `-use_encryption` flag on the command line to tell the Deploy Applications Tool that encryption is being used and to prompt for the encryption passphrase.  As with the database passwords, the tool can also read the passphrase from standard input (for example, `stdin`) to allow the tool to run without any user input. You can bypass the stdin prompt with two other options. Store the passphrase in an environment variable, and use the environment variable name with command line option -passphrase_env. Another option is to create a file with the single value of passphrase. Pass this filename using the command line option -passphrase_file
```
Comment (ddsharpe) [inline documentation/1.9/content/userguide/tools/deploy.md:38]: same as previous comment.

File: documentation/1.9/content/userguide/tools/discover.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ A command line example containing the variable file name:
 
     $ weblogic-deploy\bin\discoverDomain.cmd -oracle_home c:\wls12213 -domain_home domains\DemoDomain -archive_file DiscoveredDemoDomain.zip -model_file DiscoveredDemoDomain.json -variable_file DiscoverDemoDomain.properties
 
-To discover the domain using online WLST, simply include the admin user name and admin URL on the command line. The tool will prompt for a password to be entered into STDIN.
+To discover the domain using online WLST, simply include the admin user name and admin URL on the command line. The tool will prompt for a password to be entered into STDIN. To bypass the prompt, you can use one of two options. Store the password in an environment variable, and use the variable name with command line option -admin_pass_env. Store the password as the only variable in a file. Provide the file name with command line option -admin_pass_file.
```
Comment (ddsharpe) [inline documentation/1.9/content/userguide/tools/discover.md:30]: Replace `Store the password as the only variable in a file. ` as described above.

File: documentation/1.9/content/userguide/tools/update.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ In WLST online mode, simply add the information on how to connect to the WebLogi
 
     $ weblogic-deploy\bin\updateDomain.cmd -oracle_home c:\wls12213 -domain_type WLS -domain_home domains\DemoDomain -archive_file DemoDomain.zip -model_file DemoDomain.yaml -variable_file DemoDomain.properties -admin_url t3://127.0.0.1:7001 -admin_user weblogic
 
-As usual, the tool will prompt for the password (it can also be supplied by piping it to standard input of the tool).
+As usual, the tool will prompt for the password (it can also be supplied by piping it to standard input of the tool). To bypass the prompt, you can use one of two options. Store the password in an environment variable, and use the variable name with command line option -admin_pass_env. Store the password as the only variable in a file. Provide the file name with command line option -admin_pass_file.
```
Comment (ddsharpe) [inline documentation/1.9/content/userguide/tools/update.md:22]: file name -> filename 
and same comment for "only variable in a file"

File: documentation/1.9/content/userguide/tools/create.md (hunk 2 above)
Comment (rosemarymarano) [reply documentation/1.9/content/userguide/tools/create.md:35]: I would re-word these sentences as follows, "You can bypass the stdin prompt with two other options: store the passphrase in an environment variable, and use the environment variable name with the command-line option `-passphrase_env` or create a file with the single value of the passphrase. Provide the name of the file with the command-line option `-passphrase_file`. The passphrase will be read by the tool from the file.

File: documentation/1.9/content/userguide/tools/update.md (hunk 5 above)
Comment (rosemarymarano) [reply documentation/1.9/content/userguide/tools/update.md:22]: file name (adj) (n)
Use two words in instructional text, but use filename as a variable in code instructions and commands.

**Review by ddsharpe — APPROVED** [review]

**Review by rakillen — APPROVED** [review]

**Pull request merged**